packages/hw-app-btc/src/Btc.js

```diff
@@ -697,6 +699,32 @@ btc.createPaymentTransactionNew(
           });
         }
       })
+      .then(() => {
+        return foreach(multisigInputs, input => {
+          return doIf(!resuming, () =>
+            getTrustedInputCall(input[1], input[0]).then(trustedInput => {
```

Comment:
Don't we want to pass the additionals here?

Reply:
Good point, btw if you want to test this actually works you could try running this branch of the wallet locally https://github.com/blockchain/blockchain-wallet-v4-frontend/pull/1306. I can send you some bch as well.